libqtile/backend/x11/core.py

```diff
@@ -596,6 +618,18 @@ def handle_KeyPress(self, event) -> None:  # noqa: N802
         keysym = self.conn.code_to_syms[event.detail][0]
         self.qtile.process_key_event(keysym, event.state & self._valid_mask)
 
+    def handle_KeyRelease(self, event) -> None:  # noqa: N802
+        assert self.qtile is not None
+
+        keysym = self.conn.code_to_syms[event.detail][0]
+        # We need to fix the mask if we use a modifier key like Control_R as a
+        # regular key. Otherwise it would be reported as Ctrl+Control_R and thus
+        # be ignored as an "unknown keysym"
```

Comment:
Does this not happen with key-down bindings that use modifiers as the main key?

Reply:
I am not sure if I get your point there. I think that `Control_L` alone is reported as `Ctrl+Control_L` both when pressing as well as releasing it.